cardano-api/cardano-api.cabal

```
@@ -406,10 +408,10 @@ test-suite cardano-api-golden
     errors,
     filepath,
     hedgehog >=1.1,
-    hedgehog-extras ^>=0.6.1.0,
+    hedgehog-extras >=0.6.1 && <0.6.5.1,
```

This is not needed anymore.
```suggestion
    hedgehog-extras >=0.6.1
```